isso/js/tests/integration/puppet.test.js

```diff
@@ -264,3 +269,32 @@ test("should execute GET/PUT/POST/DELETE requests correctly", async () => {
     { text: 'New comment body' },
   );
 });
+
+test("Postbox submit button should be disabled on submit click and enabled after response", async () => {
```

Comment:
Thanks for pointing that out.

I've actually already added a check to ensure the button isn't disabled after a response in the existing test: 
"should fill Postbox with valid data and receive 201 reply." 

You can see the change here: https://github.com/isso-comments/isso/pull/993/files#diff-f636e0fc5201601aa0484b5e325aa506fae5d9b6fcba34d635497f94939ed059R151. 

Since that covers this scenario, a separate test might be redundant.

Reply:
I agree, you're covering all the bases already. Just please adjust the test titles (this one and the one above outside the diff) so that they reflect what's actually happening.